src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -207,6 +207,9 @@ class CollectedHeap : public CHeapObj<mtGC> {
   }
 
  public:
+  MemRegion reserved_region() const {
+    return _reserved;
+  }
```

Comment:
Could you extract out this (a new CollectedHeap API) to its own ticket/PR? (It seems important enough to not be hidden inside a CDS PR.)

Reply:
Sure. I will open an issue and PR for this.